packages/js/ai/src/utils/text-completion.ts

```diff
@@ -6,9 +6,15 @@ import { requestJetpackToken } from './requestJetpackToken';
 /**
  * Leaving this here to make it easier to debug the streaming API calls for now
  *
- * @param {string} prompt - The query to send to the API
+ * @param {string} prompt         - The query to send to the API
+ * @param {string} feature        - The feature to use for the completion
+ * @param {string} responseFormat - The format of the response. Can be 'text' or 'json_object'.
  */
-export async function getCompletion( prompt: string, feature: string ) {
+export async function getCompletion(
+	prompt: string,
+	feature: string,
+	responseFormat?: string
```

Comment:
See comment above: right now, getCompletion() isn't checking that responseFormat here is the correct type, but, you could import the responseFormat type and use it here:
```suggestion
	responseFormat?: ResponseFormat
```

Reply:
When that's done, if I change the default where we call `getCompletion()`, I'll get a Typescript error where it was unnoticed before:

```
			const suggestionsSource = await getCompletion(
				prompt,
				completionFeature,
				responseFormat ?? 'audio'
			);
```

> 	Argument of type 'ResponseFormat | "audio"' is not assignable to parameter of type 'ResponseFormat | undefined'.
  Type '"audio"' is not assignable to type 'ResponseFormat | undefined'